### PR TITLE
health endpoint

### PR DESCRIPTION
**Pull request by MashB**
Enable health endpoint

### CODE REVIEW

**Review by sujata-m — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,9 @@ WORKDIR /opt/keycloak
 # Copy the realm file into the appropriate directory
 COPY tdei-realm.json /opt/keycloak/data/import/tdei-realm.json
 
+#ENV used by build script to enable health endpoint 
+ENV KC_HEALTH_ENABLED=true
```
Comment (susrisha) [inline Dockerfile:10]: this is already applied on app service environment variables

Comment (MashB) [reply Dockerfile:10]: Build arguments and run arguments are different. We need this as part of build argument for packaging

File: Dockerfile [hunk 2]
```diff
@@ -6,6 +6,9 @@ WORKDIR /opt/keycloak
...
+
 RUN /opt/keycloak/bin/kc.sh build --db postgres
```
Comment (susrisha) [inline Dockerfile:12]: change command to add --health-enabled=true

**Comment by susrisha** [comment]
--health-enable is a build option. This is verified at https://www.keycloak.org/server/health as a build option. Approving it

**Review by susrisha — APPROVED** [review]

**Review by iamrajeshk — APPROVED** [review]

**Pull request merged**